text/trace/0170-sampling-probability.md

```diff
@@ -617,10 +617,201 @@ stream of Span data with non-zero values in the `log_head_adjusted_count`
 field can approximately and accurately count Spans using adjusted
 counts.
 
-Non-probabilistic Samplers such as the [Leaky-bucket rate-limited
-sampler](https://github.com/open-telemetry/opentelemetry-specification/issues/1769)
-SHOULD set the `log_head_adjusted_count` field to zero to indicate an
-unknown adjusted count.
+### Span data model changes
+
+The OpenTelemetry trace data "model" is not currently specified in a
+stand-alone way, the way it has been done for logs and metrics.  This
+OTEP calls for the creation of a new `trace/datamodel.md` file to
+capture this sort of specification detail.
+
+Addition to the Span data model:
```

Comment:
I think we are adding a lot of new "Samplers" that we had not agreed on yet. We should stick with the once we define/agreed on.

Reply:
The scope of this project is to repair the underspecified `TraceIDRatioBased` sampler. No new samplers are introduced in this proposal. Some have suggested the sampler should be renamed because it is no longer based on trace-ID-ratio, but in general the behavior is to select a ratio of trace-IDs so I think the name is fine.

Reply:
Ok.. probably overwhelmed with too many terminologies like strict vs non-strict probabilistic sampling etc. But if you think all are related to just this sample it is fine.

Reply:
OK. fwiw the word "strict" isn't used anywhere afaik. 😀

Reply:
Sorry consistent vs lack of consistent :)